test/framework/controller/installation_manager.go

```diff
@@ -24,6 +24,9 @@ import (
 type InstallationManager interface {
 	InstallCNIMetricsHelper(image string, tag string) error
 	UnInstallCNIMetricsHelper() error
+	InstallTigeraOperator(version string) error
+	UninstallTigeraOperator() error
+	//AddAndUpdateRepository(entry repo.Entry)
```

Comment:
Can we remove this commented code ?

Reply:
Thanks for catching this. Will remove it.